base_geoengine/geo_model.py

```diff
@@ -18,35 +18,42 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from __future__ import absolute_import
-import json
 
-from osv import fields, osv, orm
-from tools.translate import _
-from . import geo_field
-from . import  geo_db
+from openerp import models
+from openerp.exceptions import except_orm, MissingError
+from openerp.osv import fields
+from openerp.tools.translate import _
 from . import geo_operators
+from . import fields as geo_fields
 
 
-DEFAULT_EXTENT = '-123164.85222423, 5574694.9538936, 1578017.6490538, 6186191.1800898'
+DEFAULT_EXTENT = '-123164.85222423, 5574694.9538936, ' \
+    '1578017.6490538, 6186191.1800898'
 
-class GeoModel(orm.BaseModel):
-    #Array of ash that define layer and data to use
+
+class GeoModel(models.BaseModel):
+    """Base class for all models defining geo fields.
+    """
+
+    # Array of ash that define layer and data to use
     _georepr = []
     _name = None
     _auto = True
-    _register = False # not visible in ORM registry, meant to be python-inherited only
-    _transient = False # True in a TransientModel
+    # not visible in ORM registry, meant to be python-inherited only
+    _register = False
+    _transient = False  # True in a TransientModel
 
     def _auto_init(self, cursor, context=None):
-        ## We do this because actually creation of fields in DB is not actually
-        ## delegated to the field it self but to the ORM _auto_init function
         """Initialize the columns in dB and Create the GIST index
-        only create and update supported"""
+        only create and update supported
+
+        We override the base methid  because creation of fields in DB is not
+        actually delegated to the field it self but to the ORM _auto_init
+        function
+        """
         columns = {}
         geo_columns = {}
         tmp = {}
-        #geo_db.init_postgis(cursor)
```

Comment:
@bwrsandman on which line do you want a comment?? 
```geo_db.init_postgis(cursor)``` is removed from the code. The line no more exists.